single-user-sobigdata/tests/test.ipynb

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
the notebook test was already there (sobigdata.ipynb) we can just remove this file